app/gui/src/dashboard/layouts/CategorySwitcher/categorySwitcherLocalStorage.ts

```diff
@@ -1,10 +1,7 @@
 /** @file Local storage keys for the category switcher. */
 import { defineLocalStorageKey } from '#/providers/LocalStorageProvider'
 
-export const {
-  useGet: useGetLocalRootDirectories,
-  useSet: useSetLocalRootDirectories,
-  useState: useLocalRootDirectoriesState,
-} = defineLocalStorageKey('localRootDirectories', {
-  schema: (z) => z.string().array().readonly(),
-})
+export const { use: useLocalRootDirectories, useState: useLocalRootDirectoriesState } =
```

Comment:
Why is it still `use`? Is it a hook?

Reply:
yup. it uses `localStorage` from the context, not sure if it's necessary tbh but i think removing the `LocalStorageProvider` is out of scope for this PR?